core-bundle/src/EventListener/RequestTokenListener.php

```diff
@@ -89,4 +91,17 @@ private function getTokenFromRequest(Request $request): string|null
 
         return null;
     }
+
+    private function isSimpleCorsRequest(Request $request): bool
+    {
+        return \in_array(
+            $request->headers->get('content-type'),
+            [
+                'application/x-www-form-urlencoded',
+                'multipart/form-data',
```

Comment:
Do we need to use `str_starts_with()` here or parse the header value as the browser sends something like `Content-Type: multipart/form-data; boundary=…` as the header value. Or does Symfony already parse this and always return `multipart/form-data` here?